src/main/java/racingcar/Racing.java

```diff
@@ -1,2 +1,76 @@
-package racingcar;public class Racing {
+package racingcar;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.List;
+
+public class Racing {
+    private static final ConsoleOutputHandler outputHandler = new ConsoleOutputHandler();
+    private static final ConsoleInputHandler inputHandler = new ConsoleInputHandler();
+    private static int maxDistance;
+    private static int totalStage;
+
+    public void race() {
+
+        outputHandler.showStartComment();
+        List<String> names = inputHandler.getUserNameInput();
+        List<Player> players = names.stream()
+                .map(Player::new)
+                .toList();
+
+        outputHandler.showCommentForChoosingStageSize();
+        totalStage = inputHandler.getTotalStageSizeInput();
+
+        outputHandler.showCommentForResult();
+        startRacing(players);
+
+        List<Player> winners = findWinnerFrom(players);
+        outputHandler.showWinners(winners);
+
+    }
+
+    private static void startRacing(List<Player> players) {
+
+        for (int i = 0; i < totalStage; i++) {
+
+            checkAndMove(players);
+            outputHandler.showSnapshotOfCurrentStage(players);
+
+        }
+
```

Comment:
불필요한 빈 줄이 섞여 보이는 듯합니다. 한 번에 적어도 전혀 읽는 데 무리가 없으니 참고해 주세요 😉 
```suggestion
        for (int i = 0; i < totalStage; i++) {
            checkAndMove(players);
            outputHandler.showSnapshotOfCurrentStage(players);
        }
```